auth-web/src/components/auth/TestimonialQuotes.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section id="testimonials-container">
     <v-container class="quote-container">
-      <div class="fading">
+      <div>
```

Comment:
If this class is no longer needed, can you clean up the styles below?

Reply:
Still needed, but this was the simplest way of disabling the fade effect until the 2nd quote is provided. Soon but sounds like not before mvp.

Reply:
In that case, I recommend leaving the old code, commented out, as well as the temporary new code and a comment explaining what's going on. Just in case you get run over by a bus and someone else needs to understand what's happening here.

Reply:
Morbid but doable 👌 😆